contracts/FungibleTokenSwitchboard.cdc

```diff
@@ -89,35 +89,36 @@ pub contract FungibleTokenSwitchboard {
         /// @param paths: The paths where the public capabilities are stored.
         /// @param address: The address of the owner of the capabilities.
         /// 
-        pub fun addNewVaultsByPath(paths: [PublicPath], address: Address) {
+        access(all) fun addNewVaultsByPath(paths: [PublicPath], address: Address) {
```

Comment:
After looking at the related transactions, I feel like these privileged methods should be behind an entitlement like `FungibleTokenSwitchboard.Owner` and the unentitled `Switchboard` should be published in public. Not a blocker at all for this PR, but that approach feels more consistent with what's done in ExampleToken and the emerging best practices around entitlements.

Reply:
yes, we should use entitlements here. I'll add those